### DISCUSSION TITLE
First time User, getting error "Record of type 'a' already exists for the specified hostname"

### DISCUSSION THREAD

**Discussion by harryfine** [qdm12/ddns-updater#878] [Q&A] [answered]
Hello, and thank you for this project.  I'm in the process of setting up a domain and DDNS and getting a certificate to put Home Assistant behind, first time for me and I'm not an expert.

I installed the Linux binary, created the data folder and created the config.json file.  I got an example from your site for EasyDNS.  The config file with redactions is:

```
{
  "settings": [
    {
      "provider": "easydns",
      "domain": "harry-fine.ca",
      "username": "zzzzz_zzzz",
      "token": "xxxxxxxxxxxxxxxx",
      "ip_version": "ipv4",
      "ipv6_suffix": ""
    }
  ]
}
```

When I run it in the terminal, I get success followed by an error:
```

2024-12-09T19:28:44-05:00 INFO ipv4 address of yyyyy-yyyy.ca is 64.68.200.44 and your ipv4 address is 142.189.154.76
2024-12-09T19:28:44-05:00 INFO Updating record [domain: harry-fine.ca | owner: @ | provider: easydns | ip: ipv4] to use 142.189.154.76
2024-12-09T19:28:44-05:00 ERROR unknown response received: <html><body><font face="sans-serif" size="-1">error<br /><hr noshade size="1">record of type 'a' already exists for the specified hostname<br /></font></body></html>
```

Can you put me in the right direction.  Also very happy to donate to the project.

Thank you

Harry Fine
Toronto, Canada

**Comment by qdm12** [comment]
I'm not sure, checking the code it doesn't even attempt to create a record (only update). Maybe @charlydaily do you have an idea (original author of #480)?

**Comment by harryfine** [comment, marked as answer]
It was my issue, I had a record set up in my zone that should not have been there. It now works perfectly.

**Reply by qdm12** [reply]
What record if I may ask? It's strange it would conflict still I think.
